ontology.ttl

```diff
@@ -576,11 +525,6 @@ foaf:logo rdf:type owl:AnnotationProperty .
                 rdfs:subClassOf :DataProduct .
 
 
-###  http://odahub.io/ontology#POSIXPath
-:POSIXPath rdf:type owl:Class ;
-           rdfs:subClassOf :FileReference .
-
-
```

Comment:
Why this and other removals? Did you merge `main` properly?

Reply:
Indeed, something went wrong during merge. I have recovered the changes from main branch